homework/tests/HomeworkTests.spec.js

```diff
@@ -0,0 +1,132 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+const data = [
+  'Prototype',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9'
+]
+
+
+data.forEach(version => {
+
+
+test.describe(version + ': Add', () => {
+    test('Adding 2 and 3 results in 5', async ({ page }) => {
+      await page.goto('https://testsheepnz.github.io/BasicCalculator');
+      await page.selectOption('#selectBuild', { label: version});
```

Comment:
instead of writing the same selectors and link every time, you could move them to variables, define it once and everywhere else use the location as variable.